### DISCUSSION TITLE
Albert skal sove mere og arbejde mindre om natten

### DISCUSSION THREAD

**Discussion by arnenator** [BDSA-2021/BDSA21-Say-AAAAAA#67] [General]
Vi har et issue med at Joachim tager alt Alberts søvn. Det er ikke godt. Der skal være en balance. Albert og Joachim skal nogenlunde sove lige meget.

**Comment by avborup** [comment]
There is an imbalance in the force... >:(

**Comment by JoachimBorup** [comment]
Vi udligner hinanden. Ikke et problem
